acapy_agent/did/cheqd/cheqd_manager.py

```diff
@@ -107,3 +116,142 @@ async def register(self, options: dict) -> dict:
             "did": did,
             "verkey": verkey,
         }
+
+    async def update(
+        self, did: str, didOperation: str, didDoc: dict, options: dict
+    ) -> dict:
+        """Update a Cheqd DID."""
+
+        async with self.profile.session() as session:
+            try:
+                wallet = session.inject(BaseWallet)
+                if not wallet:
+                    raise web.HTTPForbidden(reason="No wallet available")
+
+                # Check if did is valid
+                await self.resolver.resolve(self.profile, did)
```

Comment:
we need to handle another condition here

after resolving the document, check if the did is deactivated?. Deactivated DIDs can be updated